Kickstarter-iOS/Views/Cells/ThanksCategoryCell.swift

```diff
@@ -3,14 +3,48 @@ import Library
 import Prelude
 import UIKit
 
+internal protocol ThanksCategoryCellDelegate: AnyObject {
+  func thanksCategoryCell(_ cell: ThanksCategoryCell, didTapSeeAllProjectsWith category: KsApi.Category)
+}
+
 internal final class ThanksCategoryCell: UITableViewCell, ValueCell {
+  internal weak var delegate: ThanksCategoryCellDelegate?
+  fileprivate let viewModel: ThanksCategoryCellViewModelType = ThanksCategoryCellViewModel()
+
   @IBOutlet fileprivate var seeAllProjectsButton: UIButton!
 
+  override func awakeFromNib() {
+    super.awakeFromNib()
+
+    self.seeAllProjectsButton
+      .addTarget(self, action: #selector(self.seeAllProjectsButtonTapped), for: .touchUpInside)
+  }
+
   func configureWith(value category: KsApi.Category) {
+    self.viewModel.inputs.configureWith(category: category)
+  }
+
+  internal override func bindStyles() {
+    super.bindStyles()
+
     _ = self.seeAllProjectsButton
       |> greyButtonStyle
-      |> UIButton.lens.title(for: .normal) %~ {
-        _ in Strings.See_all_category_name_projects(category_name: category.name)
+  }
+
+  internal override func bindViewModel() {
+    super.bindViewModel()
+
+    self.viewModel.outputs.notifyDelegateToGoToDiscovery
+      .observeForControllerAction()
+      .observeValues { [weak self] in
+        guard let _self = self else { return }
```

Comment:
By the way, Swift now lets us do `guard let self = self else { ... }` for this sort of thing.